client/src/components/ReviewList.js

```diff
@@ -1,11 +1,23 @@
 import React from "react";
 import ReviewTile from "./ReviewTile.js";
 
-const ReviewList = (props) => {
+const ReviewList = ({ reviews, handleVote }) => {
   let reviewTiles = ""
-  reviewTiles = props.reviews.map(review => {
-    return <ReviewTile key={review.id} review={review} />
-  })
+  const sortedReviews = reviews.sort((a,b) => {
+    if ( a.votes < b.votes ){
+      return 1
+    }
+    if ( a.votes > b.votes ){
+      return -1
+    }
+    return 0
+  }) 
```

Comment:
data manipulation belongs on the backend - the router should handle any sorting before data is passed to the front end